sql/src/test/java/io/crate/execution/engine/window/WindowDefinitionTest.java

```diff
@@ -58,6 +58,66 @@ public void prepare() throws IOException {
             .build();
     }
 
+    @Test
+    public void test_offset_preceding_does_not_support_interval_in_rows_mode() {
+        expectedException.expectMessage(containsString("In ROWS mode the offset must be a non-null, non-negative number"));
+        e.analyze(
+            "select id, avg(date::long) over(twenty_sec_prior)" +
+            "  from doc.users" +
+            "  window twenty_sec_prior as (rows between '20 seconds'::interval preceding and current row)" +
+            "  order by id");
+    }
+
+    @Test
+    public void test_offset_preceding_does_not_support_negative_values_in_rows_mode() {
+        expectedException.expectMessage(containsString("In ROWS mode the offset must be a non-null, non-negative number"));
+        e.analyze(
+            "select id, avg(date::long) over(twenty_sec_prior)" +
+            "  from doc.users" +
+            "  window twenty_sec_prior as (rows between -2000 preceding and current row)" +
+            "  order by id");
+    }
+
+    @Test
+    public void test_offset_following_does_not_support_interval_in_rows_mode() {
+        expectedException.expectMessage(containsString("In ROWS mode the offset must be a non-null, non-negative number"));
+        e.analyze(
+            "select id, avg(date::long) over(twenty_sec_following)" +
+            "  from doc.users" +
+            "  window twenty_sec_following as (rows between current row and '20 seconds'::interval following)" +
+            "  order by id");
+    }
+
+    @Test
+    public void test_offset_following_does_not_support_negative_values_in_rows_mode() {
+        expectedException.expectMessage(containsString("In ROWS mode the offset must be a non-null, non-negative number"));
+        e.analyze(
```

Comment:
These kind of semantic / error handling tests don't really have anything to do with adding interval support.

Can we revert this from this PR and if they're useful to add, do that in a separate PR? (I thought we cover this already elsewhere)

Reply:
remove them, ok